restli-server/src/main/java/com/linkedin/restli/internal/server/response/ResponseUtils.java

```diff
@@ -252,16 +252,17 @@ public static RestException buildRestException(RestLiResponseException restLiRes
     return new RestException(restResponse, cause==null ? null : cause.toString(), cause, writableStackTrace);
   }
 
-  public static StreamException buildStreamException(RestLiResponseException restLiResponseException, StreamDataCodec codec)
+  public static StreamException buildStreamException(RestLiResponseException restLiResponseException, ContentType contentType)
```

Comment:
since this is a public interface we should not update the interface.
can add the header in RestLiResponseException.restliResponse.headers before building stream exception response